cinema/views.py

```diff
@@ -21,22 +30,33 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class ListCreateModelsViewMixin(
```

Comment:
The new class is not a mixin since it inherits from `GenericViewSet` as well. I'll suggest removing this class and inherit all views from necessary mixins explicitly to provide better customization in the future